src/App.tsx

```diff
@@ -15,71 +20,91 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
-
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-
-      {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
-
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+export class App extends Component<{}, State> {
+  state: State = {
+    selectedGood: 'Jam',
+  };
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+  clearStateSelectedGood = () => {
+    this.setState({ selectedGood: '' });
+  };
 
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
-            <button
-              data-cy="RemoveButton"
-              type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
+  render() {
+    const { selectedGood } = this.state;
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Jam
-          </td>
-        </tr>
+    return (
+      <main className="section container">
+        {
+          selectedGood
+            ? (
+              <h1 className="title is-flex is-align-items-center">
+                {`${selectedGood} is selected`}
+                {
+                  selectedGood
+                      && (
+                      /* eslint-disable-next-line jsx-a11y/control-has-associated-label */
+                        <button
+                          data-cy="ClearButton"
+                          type="button"
+                          className="delete ml-3"
+                          onClick={this.clearStateSelectedGood}
+                        />
+                      )
+                }
+              </h1>
+            )
+            : <h1 className="title">No goods selected</h1>
+        }
+        <table className="table">
+          <tbody>
+            {
+              goods.map(good => (
+                <tr
+                  // eslint-disable-next-line max-len
+                  className={classNames({ 'has-background-success-light': selectedGood === good })}
+                  data-cy="Good"
+                  key={good}
+                >
+                  <td>
+                    {
+                      (selectedGood === good)
+                        ? (
```

Comment:
Here the same. You can make code with ternary operator shorter
```suggestion
                    {(selectedGood === good) ? (
```